mkdocs.redirects.yml

```
@@ -126,7 +126,6 @@ plugins:
       Reference/Resources.md: index.md
       HowTo/Use-Privacy/EEA-compliant.md: private-networks/how-to/use-privacy/eea-compliant.md
       HowTo/Use-Privacy/Privacy.md: private-networks/how-to/use-privacy/besu-extended.md
-      HowTo/Use-Privacy/Use-GoQuorum-compatible-privacy.md: private-networks/how-to/use-privacy/goquorum-compatible.md
       HowTo/Use-Privacy/Run-Tessera-With-Besu.md: private-networks/how-to/use-privacy/tessera.md
```

```suggestion
      HowTo/Use-Privacy/Use-GoQuorum-compatible-privacy.md: private-networks/concepts/privacy/index.md
      private-networks/how-to/use-privacy/goquorum-compatible.md: private-networks/concepts/privacy/index.md
      HowTo/Use-Privacy/Run-Tessera-With-Besu.md: private-networks/how-to/use-privacy/tessera.md
```

Deleting a page requires adding a redirect - I just redirected to the main privacy concepts page for now. Deleted redirects also must be replaced.